okonomiyaki/file_formats/setuptools_egg.py

```diff
@@ -74,37 +79,84 @@ def _get_default_setuptools_abi(platform_string, pyver):
             raise ValueError(msg)
 
 
+class _UnspecifiedArg(object):
+    pass
+_UNSPECIFIED = _UnspecifiedArg()
```

Comment:
I think you probably don't need a new class for this, just an instance of `object`.